lib/rspec/core/configuration.rb

```diff
@@ -142,7 +142,7 @@ def expose_dsl_globally=(value)
 
       # Determines where deprecation warnings are printed.
       # Defaults to `$stderr`.
-      # @return [IO] IO to write to
+      # @return [IO, String] IO to write to or filename to write to
```

Comment:
How about "IO or filename to write to" instead of "IO to write to or filename to write to" ?